### ISSUE TITLE
Version 2.2.8 Causes Blue Screen of Death

### ISSUE THREAD

**Issue by AnyHoLiC-F** [Ryochan7/DS4Windows#1918]
### **Description**
Using version 2.2.8, when I turn on the controller while a game is running, Windows 10 crashes and blue screen of death turns up. It shows 
> stop code: IRQL NOT LESS OR EQUAL

I was able to recreate the scenario five times after the original one happened (6 total).

**Additional Info:**
I tried to find proper logs at the exact time of issue, but nothing to be found (Not Windows nor DS4W).

This happened while playing Destiny 2 and Outriders Demo on Steam.

I played Outriders without DS4Windows 2.2.8 (using Steam built-in support) and everything was fine, no blue screen.

Now I reverted back to version 1.7.28 _(one of the most stable ones, to me at least)_. Started DS4Windows, then Destiny 2. I turned on the controller while the game is running and Windows didn't crash.

**Desktop:**
 - Controller Make and Model: Sony DS4 v.1 (CUH-ZCT1E)
 - OS: [Windows 10 Pro Build 19042 (Version: 20H2, 19041.vb_release.191206-1406)]
 - DS4Windows Version [2.2.8]

**Comment by AnyHoLiC-F** [comment]
Problem seems to be caused by **steamxbox.sys**.
Stop code

> DRIVER IRQL NOT LESS OR EQUAL

**Comment by Masamune3210** [comment]
That driver is Steam's seemingly extremely unstable Enchanced Xbox Controller driver, which I have seen do nothing but interfere and conflict with things, causing bugchecks along the way

**Comment by AnyHoLiC-F** [comment]
@Masamune3210 
It only appeared as the issue after the last blue screen. The previous ones didn't indicate the issue.
I uninstalled Steam. I will remove the controller from devices next, then I will restart and install everything. I will make sure that controller support in Steam big picture is disabled for all.

I will post the result here later.

**Comment by AnyHoLiC-F** [comment]
@Masamune3210 
> Solution: Use **Autoruns** to remove or disable broken drivers ".sys".

After almost breaking my Windows and PC, I stumbled upon a proper solution which seemed to work perfectly fine for now.

Apparently, there's an official tool/utility from Microsoft called **Autoruns** which lets you disable or remove drivers. It has many other features too.
It did the trick for me. I disabled the driver simply by unchecking a box next to "steamxbox.sys" while running the tool in admin mode. I played Destiny 2 for 20 mins without a blue screen.

I tested it while using DS4W v1.7.28. Maybe I'll try the latest version later.
